src/Devices/JSONBuilder.cpp

```diff
@@ -56,31 +56,32 @@ int JSONBuilder::buildCurrentValues() {
   uint16_t minutes = (ms - (days * 86400000) - (hours * 3600000)) / 60000;
   uint16_t seconds = (ms - (days * 86400000) - (hours * 3600000) - (minutes * 60000)) / 1000;
 
```

Comment:
I don't understand how the formatting rules liked one then the other, but at least we are following some automated rules!